frontend/src/App.css

```diff
@@ -22,6 +22,74 @@
   text-align: center;
 }
 
+/*Page Home*/
+h2 {
+  text-align: center;
+  color: #dec439;
+  padding: 10%;
+  margin-top: 6rem;
+}
+
+/*Page SignIn*/
+
+.bouton {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+}
+
+.link {
+  display: flex;
+  justify-content: space-around;
+  font-size: 1.5rem;
+}
+
+.PresInscip {
+  text-align: center;
+  padding: 5rem;
+}
+
+.Inscription {
+  text-align: justify;
+  line-height: 5;
+  padding-left: 35%;
```

Comment:
small padding all around, or center content, but padding-left: 35% is a bad practice